app/views/articles/_badges_area.html.erb

```diff
@@ -46,7 +46,7 @@
             src="<%= cloudinary achievement.badge.badge_image_url, 180 %>"
             alt="<%= achievement.badge.title %>"
             title="<%= achievement.badge.title %>"
-            class="badge-achievement-image"
+            class="badge-achievement-image mx-auto d-block"
```

Comment:
We do not have `d-block` utility class. For `display: block;` we have a class `block` (so without the prefix `d-`).

Reply:
Hello How I can make this change ? It's my first contribute ever :) , Should I make the edit to the code again and commit again ? or just edit and push and request another pull request.

Reply:
Yeap, you can edit code on current branch - you don't have to create another PR.

Reply:
Done

Reply:
But because `d-block` doesn't exist it actually didn't apply any extra styling so you probably don't need `block` as well, right?

Reply:
yea I check it , it useless here should I remove it?